result/flow.yaml

```diff
@@ -190,6 +190,56 @@ components:
           items:
             $ref: '#/components/schemas/Flow.TaggedMetric'
           x-field-uid: 15
+        tx_l1_rate_bps:
+          description: >-
+            Layer1 Tx frame rate in bits per second.
+          type: number
+          x-field-uid: 16
+        rx_l1_rate_bps:
+          description: >-
+            Layer1 Rx frame rate in bits per second.
+          type: number
+          x-field-uid: 17
+        tx_rate_bytes:
+          description: >-
+            Layer1 Tx frame rate in bytes per second.
+          type: number
+          x-field-uid: 18
+        rx_rate_bytes:
+          description: >-
+            Layer1 Tx frame rate in bytes per second.
+          type: number
+          x-field-uid: 19
+        tx_rate_bps:
+          description: >-
+            Layer1 Tx frame rate in bits per second.
```

Comment:
What is the difference between this and tx_l1_rate_bps . Currently descriptions are identical which does not look correct.